__tests__/__unit__/shared/utils.unit.test.ts

```diff
@@ -0,0 +1,128 @@
+/*
+* This program and the accompanying materials are made available under the terms of the *
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at *
+* https://www.eclipse.org/legal/epl-v20.html                                      *
+*                                                                                 *
+* SPDX-License-Identifier: EPL-2.0                                                *
+*                                                                                 *
+* Copyright Contributors to the Zowe Project.                                     *
+*                                                                                 *
+*/
+
+import * as utils from "../../../src/utils";
+import * as sharedUtils from "../../../src/shared/utils";
+import * as globals from "../../../src/globals";
+import { ZoweDatasetNode } from "../../../src/dataset/ZoweDatasetNode";
+import * as vscode from "vscode";
+import * as path from "path";
+import {
+    generateIProfile,
+    generateISessionWithoutCredentials
+} from "../../../__mocks__/generators/shared";
+import { generateDatasetSessionNode } from "../../../__mocks__/generators/datasets";
+
+describe("Add filterTreeByString Tests", () => {
```

Comment:
I thought we agreed to have this in every _describe_ block:
`afterEach(() => { jest.clearAllMocks(); });`

This should be included in every _describe_ block to prevent tests from contaminating each other.

Even if there is only one _it_ block, this should be present in case someone adds more tests later.